### ISSUE TITLE
sqlalchemy example should just use cockroachdb and not postgresql in the connection string

### ISSUE THREAD

**Issue by dikshant** [cockroachdb/docs#15992]
I was a little confused by this example in our sqlalchemy docs and saw a customer getting indirectly confused due to this.
```python3
if __name__ == '__main__':
    # For cockroach demo:
    # DATABASE_URL=postgresql://demo:<demo_password>@127.0.0.1:26257?sslmode=require
    # For CockroachCloud:
    # DATABASE_URL=postgresql://<username>:<password>@<globalhost>:26257/<cluster_name>.defaultdb?sslmode=verify-full&sslrootcert=<certs_dir>/<ca.crt>
    db_uri = os.environ['DATABASE_URL'].replace("postgresql://", "cockroachdb://")
    try:
```
I don’t see a good reason to do this string substitution here. We should just have:
```python3
if __name__ == '__main__':
    # For cockroach demo:
    # DATABASE_URL=cockroachdb://demo:<demo_password>@127.0.0.1:26257?sslmode=require
    # For CockroachCloud:
    # DATABASE_URL=cockroachdb://<username>:<password>@<globalhost>:26257/<cluster_name>.defaultdb?sslmode=verify-full&sslrootcert=<certs_dir>/<ca.crt>
    db_uri = os.environ['DATABASE_URL']
    try:
```
Because the UI already shows `cockroachdb://`
<img width="556" alt="Screenshot 2023-01-12 at 2 09 15 PM" src="https://user-images.githubusercontent.com/9084804/212159927-e7747e41-b311-4b2f-b2c6-29f1c9ff790e.png">

Also can we make it explicitly clear in a highlighted that when using SQLAlchemy the connection string should start with `cockroachdb` and not `postgres`.
See: https://github.com/cockroachdb/sqlalchemy-cockroachdb#install-and-usage

**Comment by ianjevans** [comment]
There are two reasons why we decided to modify the connection URL in the example.

1. SQL Alchemy requires a `cockroachdb` prefix on the URL to use the correct provider. You'll get errors if your connection string starts with `postgresql`.
2. All the connection URLs that the user can copy/paste, either in the Cloud Console or when starting with `cockroach demo`, are prefixed with `postrgresql`.

So we had the choice of either making the user hand-edit the copied connection string, or just handle it in code. For simplicity we handled it in code.

**Comment by dikshant** [comment]
Yep that makes sense. For now as we discussed on Slack, let's use add a notice to make it more clear that cockroachdb should be use on the connection string on the doc.
